### ISSUE TITLE
Do users created via graph need `identities` property?

### ISSUE THREAD

**Issue by rhafer** [owncloud/ocis#9883]
This is about how deal with the `identities` property of user objects  on the `graph` API.  The property is supposed to contain a collection of external  identities by which a user is know. We currently set in only in a limited amount of cases:

- Users created via Autoprovisioning:
  - The issuer is set to the value of the `iss` claim from the IDP's userinfo response
  - The `issuerAssignedId` is set to the value of the `sub` claim from the IDP's userinfo response
- Federated users accepted via ocm:
  - The `issuer` is to the `idp` value of the cs3 UserID of that user
  - The `issuerAssignedId` is to the `opaqueId` value of the cs3 UserID
 - The demo users have it set via their LDAP attributes:
   -  `issuer` is set based on OCIS_URL / OCIS_IDP_URL
   -  `issuerAssignedId` is to the userid

It's not really defined you if and how we set it for
- users created via on the graph API:
  - when an `identities` value is present in the Request body we just take that, without any validation
  - when no `identities` value is present we don't set any default.
- users coming from an external LDAP:
 - There is currently no way to configure a custom LDAP attribute to use for the `identities` value. It is hardcoded to `oCExternalIdentity`. (Probably not needed anyway, because it's such an ocis specific attribute that no existing  external read-only LDAP server can provide any useful value for it)
 - We don't set an implicit default when return LDAP users that don't have a `oCExternalIdentity`  value

There's couple of options:

1. It's probably ok if users don't have the `identities` property. We mainly need it for being able to track external users (federated or autoprovisioned). In that case we should probably remove the property again from the demo users.
2. Return an implicit `identities` property for users that don't have a value in the backend to make sure every user has `identities` set. (Base on the userid and the IDP_URL)

Long story short: I am leaning towards 1. As there is no good way to set `identities` to a useful value anyways when creating a new user (outside autoprovisioning). The `issuer` could be the url of the IDP. but `issuerAssignedId` would need to be the `sub` claim, which we don't know when creating the user.
